lib/rules/require-capitalized-constructors.js

```diff
@@ -11,6 +11,9 @@
  *
  * ```js
  * "requireCapitalizedConstructors": true
+ * "requireCapitalizedConstructors": [
```

Comment:
As per https://github.com/jscs-dev/node-jscs/issues/698, it might be more appropriate for this to be `"requireCapitalizedConstructors": { allExcept: ['somethingNative', 'anotherNativeThing'] }`